content/changelog/2025-05-16.md

```diff
@@ -0,0 +1,120 @@
+---
+
+## Introducing the Neon Data API (Early Access)
+
+We're excited to introduce the **Neon Data API**, now available in Early Access. This feature provides a RESTful API on top of your Neon Postgres database, powered by [PostgREST](https://postgrest.org). It enables direct access to tables, views, and functions using standard HTTP verbs—ideal for client-side applications and rapid prototyping.
+
+With the Neon Data API, you can:
+
+- Query tables with HTTP requests
+- Use PostgREST-compatible SDKs like `postgrest-js`
+- Scale the API alongside your Neon database
+- Secure access with Neon Auth or bring your own JWKS (e.g., Auth0, Clerk)
+
+**Enabling the Data API**  
+You can enable the API from the Neon Console or programmatically via our new `POST /data-api` endpoint. Each branch can have its own API instance.
+
+Example request:
+
+```bash
+curl --location 'https://console.neon.tech/api/v2/projects/<project_id>/branches/<branch_id>/data-api'
+  --header 'Authorization: Bearer <token>'
+  --data '{ "database_name": "mydb" }'
+```
+
+**Authentication**  
+By default, Neon Auth is provisioned with your API. You can also configure your own `jwks_url` for third-party auth providers:
+
+```bash
+curl --location 'https://console.neon.tech/api/v2/projects/<project_id>/branches/<branch_id>/data-api'
+  --header 'Authorization: Bearer <token>'
+  --data '{ "jwks_url": "https://your-auth.com/.well-known/jwks.json" }'
+```
+
+**Getting started**  
+Once enabled, your REST endpoint is ready in about 15 seconds. Here's a sample request:
+
+```bash
+curl --location 'https://<your-api-url>/playing_with_neon'
+     --header 'Accept: application/json'
+     --header 'Authorization: Bearer <jwt>'
+```
+
+**Example with postgrest-js**
+
+```javascript
+import { PostgrestClient } from '@supabase/postgrest-js'
+
+const client = new PostgrestClient('https://<your-api-url>', {
+  headers: { Authorization: 'Bearer <jwt>' }
+})
+
+const { data } = await client
+  .from('playing_with_neon')
+  .select('*')
+  .gte('value', 0.5')
+
+console.table(data)
+```
+
+Learn more in our [Data API documentation](/docs/guides/data-api).
+
+## New credit system
+
+We’ve launched a new credit system in the Neon Console that we plan to use for future promotions, referrals, and goodwill credits. To kick things off, we're offering a $20.00 credit when you upgrade to a paid Neon plan.
+
+[Claim your $20 credit](https://t.co/RryvWZIxWX)
+
+The credit will appear at the top of the Neon console and will be automatically applied to your account when you upgrade.
+
+![Credit system](/docs/relnotes/credit_system.png)
+
+## TBD: Postgres version updates
+
+We updated supported Postgres versions to [14.18](https://www.postgresql.org/docs/release/14.19/), [15.13](https://www.postgresql.org/docs/release/15.13/), [16.9](https://www.postgresql.org/docs/release/16.9/), and [17.5](https://www.postgresql.org/docs/release/17.5/), respectively.
```

Comment:
Point first link to `/14.18/` instead of `/14.19/`

Reply:
Thanks, Philip. Had to remove the entry. The new versions are not on prod yet. We'll stash this oen for next week.